include/simdjson/document.h

```diff
@@ -143,7 +145,11 @@ class array : protected internal::tape_ref {
    * Part of the std::iterable interface.
    */
   inline iterator end() const noexcept;
-
+  /**
```

Comment:
Super super minor nit: let's try and keep the *'s aligned the same way in different parts. I don't care which alignment we use. :)

Reply:
Fixed.